### PR TITLE
show local mode message conditional in convert.go

### PR DESCRIPTION
**Pull request by xilabao**
**What this PR does / why we need it**:
don't show 'running in local mode...' if the output format is specified, so we can output to a file and not need any changes.

**Which issue this PR fixes**: 

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/kubectl/cmd/convert.go [hunk 1]
```diff
@@ -152,7 +156,6 @@ func (o *ConvertOptions) Complete(f cmdutil.Factory, out io.Writer, cmd *cobra.C
 
 	// build the printer
 	o.out = out
-	outputFormat := cmdutil.GetFlagString(cmd, "output")
 	templateFile := cmdutil.GetFlagString(cmd, "template")
 	if len(outputFormat) == 0 {
```
Comment (alexandercampbell) [inline pkg/kubectl/cmd/convert.go]: Does this check need to be adjusted here?
`len(outputFormat) == 0` is now being checked twice in this function.

Comment (xilabao) [inline pkg/kubectl/cmd/convert.go]: Done. PTAL

**Pull request closed without merging**